core/templates/expressions/expression-syntax-tree.service.spec.ts

```diff
@@ -67,6 +67,11 @@ describe('Expression syntax tree service', () => {
 
     it('should extend ExpressionError class', () => {
       const exampleVar = undefined;
+      // This throws "Argument of type 'undefined' is not assignable
+      // to parameter of type 'string'." We need to suppress this
+      // error because of the need to test validations when the
+      // variable is not defined.
+      // @ts-ignore
```

Comment:
If it is not a valid input type, then it should not be tested. If it is tested, then it should be a part of typedef of that argument.

Reply:
done